pkg/utils/utils.go

```diff
@@ -257,3 +268,121 @@ func DeleteDoubleWhiteSpace(slice []string) []string {
 	}
 	return result
 }
+
+// ExtractAnnotaions returns annotations of InvolvedObject for the given event
+func ExtractAnnotaions(obj *apiV1.Event) map[string]string {
```

Comment:
This works for object which are deleted ?
We may try to extract embedded object which is delete ?

Reply:
> We may try to extract embedded object which is deleted?

I would like to discuss the workflows here

In general, 
- we use ExtractAnnotaions only for object kind: Events
- Kubernetes events of type Normal/Warning are associated with an existing object
- In most cases, we can get those objects associated with events with no trouble.
- In case of error during the getting these objects, notifs will be sent to the default channel.

In this rare case, 
- ExtractAnnotaions() may try to get the particular deleted object which results in `Error from server (NotFound): <object> "foo" not found`
- ExtractAnnotaions() then informs the error and returns empty map
- No annotations from actual objects get updated in the current event object
- botkube sends notif to the default channel or We receive event notifications even after disabling it through annotations.

Reply:
Explanation looks good to me. Lets make sure we have unit tests around this scenario.